### PR TITLE
changed action regulation to process regulation per #178

### PR DESCRIPTION
**Pull request by cameronmore**

### CODE REVIEW

**Review by neilotte — CHANGES_REQUESTED** [review]
@cameronmore It appears 'the label for 'Action Permission' was not updated.

**Comment by neilotte** [comment]
@cameronmore The definitions of permits, prohibits, requires, and their inverses still reference regulating the involvement of an agent in each process, which means these definitions still are describing acts rather than processes per se. I recommend this language also be removed so that, for instance, mere processes not obviously involving agents as causal actors (e.g. the states of affairs of a cyber system) may be prohibited, permitted, and required.

File: src/cco-extensions/ModalRelationOntology.ttl [hunk 1]
```diff
@@ -2268,7 +2268,7 @@ mro:is_permitted_by rdf:type owl:ObjectProperty ;
                     owl:inverseOf mro:permits ;
                     rdfs:domain obo:BFO_0000015 ;
                     rdfs:range cco:ActionRegulation ;
-                    cco:definition "y is_permitted_by x at t iff: x is an instance of Action Regulation at time t, and y is an instance of Act at time t, and x prescribes that some agent may be agent in y."@en ;
+                    cco:definition "y is_permitted_by x at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that some independent continuant (but not spatial regiom) may participate in y."@en ;
                     cco:is_curated_in_ontology <http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ModalRelationOntology> ;
```
Comment (neilotte) [inline src/cco-extensions/ModalRelationOntology.ttl:2272]: @cameronmore 'region' has a mispelling. 

Also, what do you think of: "y is_permitted_by x at t iff: x is an instance of Process Regulation at time t, and y is an instance of Process at time t, and x prescribes that y may occur." 

If acceptable, this phrasing would apply to multiple object properties in this PR.

**Review by neilotte — CHANGES_REQUESTED** [review]

**Comment by mark-jensen** [comment]
@cameronmore MRO is built automatically as part of the release process. There is no need to make changes manually, unless you can guarantee that every change to any property in CCO from v1.5 to v1.6 is accounted for.

**Review by mark-jensen — APPROVED** [review]

**Review by APCox — APPROVED** [review]

**Review by neilotte — APPROVED** [review]

**Pull request merged**